sim/team-validator.ts

```diff
@@ -1286,6 +1286,10 @@ export class TeamValidator {
 			}
 		}
 
+		if (dex.species.get(species.baseSpecies).tags?.length) {
```

Comment:
No need for `?` here as tags is always defined

Reply:
Sorry I have a habit of including redundant failsafes

Reply:
(You forgot to fix this...)

```suggestion
		if (dex.species.get(species.baseSpecies).tags.length) {
```

Reply:
Also you should really search the `tierSpecies` tags list.